scripts/run.sh

```diff
@@ -17,8 +17,8 @@ fi
 
 
 profile=debug
```

Comment:
@yihau do we ever run this in debug mode? Thoughts on having a separate PR that just changes the default to release mode?

I've never (intentionally) run these in debug mode

Reply:
tbh, I'm not sure if anyone is still using 🙈  just checked the git history: https://github.com/solana-labs/solana/pull/6945

yeah, I think we can have a PR to change the default behavior